tools/common/src/main/java/org/apache/cxf/tools/common/ClassUtils.java

```diff
@@ -97,8 +99,9 @@ public void compile(ToolContext context) throws ToolException {
 
                                 try {
                                     Files.copy(otherFile,
-                                        Files.createDirectories(Paths.get(targetDir, dirName)).resolve(str));
-
+                                            Files.createDirectories(Paths.get(targetDir, dirName)).resolve(str));
+                                } catch (FileAlreadyExistsException existsException) {
+                                    LOG.log(Level.WARNING, "EXIST_RESOURCE_FILE", otherFile);
```

Comment:
@jimma seems the message is not referring to the correct file: `otherFile` is a source and should exist. The target is `Files.createDirectories(Paths.get(targetDir, dirName)).resolve(str)` (probably needs a new variable)

Reply:
@reta  You are right and fixed.Thanks for the review.